### PR TITLE
rename Arduino-SignatureRow to Arduino-Signature

### PR DESCRIPTION
**Pull request by nkaaf**
I have renamed my repository because the term "SignatureRow" for AVR microcontrollers is not used by all documentations. In the documentation of the ATmega328 this section of the memory is called "Signature Row", however in the documentation of the ATtiny848 it is called "Signature Imprint Table".

Therefore is generalize the name of my repository.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @nkaaf. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done:
```
2022/09/27 12:50:57 Loaded 5177 libraries from DB
Changing URL of library SignatureRow from https://github.com/nkaaf/Arduino-SignatureRow.git to https://github.com/nkaaf/Arduino-Signature.git
Success!
```

**Pull request merged**
